dotman-plugin/src/main/java/net/minevn/dotman/UpdateChecker.kt

```diff
@@ -0,0 +1,55 @@
+package net.minevn.dotman
+
+import net.minevn.dotman.utils.Utils.Companion.info
+import net.minevn.dotman.utils.Utils.Companion.warning
+import net.minevn.libs.bukkit.parseJson
+import net.minevn.libs.get
+import org.bukkit.command.CommandSender
+
+class UpdateChecker {
+    companion object {
+        private val plugin = DotMan.instance
+        private val language = plugin.language
+        private val url = "https://api.github.com/repos/MineVN/DotMan/releases/latest"
+        private var releaseVersion : String = ""
+        private val currentVersion = plugin.description.version.trim()
+        private var latestVersion = currentVersion
+        private var latest = false
+
+        fun init() {
+            latest = checkUpdate()
+            if (latest) {
+                info(this.language.updateLatest)
+            } else {
+                language.updateAvailable
+                    .replace("%NEW_VERSION%", "")
+                    .replace("%CURRENT_VERSION%", currentVersion)
+                    .let { info(it) }
+                info(this.language.updateAvailableLink.replace("%URL%", releaseVersion))
+            }
```

Comment:
Đoạn này em có thể rút ngắn thành `loginCheckForUpdates(plugin.server.consoleSender)`

Ngoài ra em điều chỉnh các điểm sau:
- Đổi tên hàm `loginCheckForUpdates` thành `checkForUpdate`
- Đối với `info(this.language.updateLatest)` thì thêm tham số `nofifyLatestVersion: Boolean` vào hàm `checkForUpdate`

Cái này không quan trọng lắm nhưng mà em chỉnh cho quen, rèn luyện tư duy coding & tránh lặp code.

Reply:
Em đã chỉnh sửa, tuy nhiên sử dụng receiver.sendMessage() thay vì info()